ci/gpu/build.sh

```diff
@@ -58,14 +58,14 @@ gpuci_mamba_retry install -y \
                "cugraph=$MINOR_VERSION.*" \
                "cuspatial=$MINOR_VERSION.*" \
                "dask-cudf=$MINOR_VERSION.*" "dask-cuda=$MINOR_VERSION.*" \
-               "numba>=0.51.2" \
+               "numba>=0.54" \
                "bokeh>=2.3.2,<=2.4" \
                "rapids-build-env=$MINOR_VERSION.*" \
                "rapids-notebook-env=$MINOR_VERSION.*"
 
 # https://docs.rapids.ai/maintainers/depmgmt/ 
-# conda remove --force rapids-build-env rapids-notebook-env
-# conda install "your-pkg=1.0.0"
+conda remove --force rapids-build-env rapids-notebook-env
+conda install -y "holoviews>1.14.1, <=1.14.6" "datashader>0.12, <=0.13"
```

Comment:
Is this PR ready for review? If so, these lines will need to be re-commented out and we'll have to get all of the dependency versions updated in the `integration` repo also (i.e. [here](https://github.com/rapidsai/integration/blob/ae421a4f919b3f84f8f5596bf5288769e8d4fa48/conda/recipes/versions.yaml)).

Reply:
thanks @ajschmidt8, I reverted the change in this file and created a PR on the integration repo: https://github.com/rapidsai/integration/pull/407